vncviewer/Viewport.cxx

```diff
@@ -598,13 +600,40 @@ int Viewport::handle(int event)
   case FL_DRAG:
   case FL_MOVE:
   case FL_MOUSEWHEEL:
-    buttonMask = 0;
+
+    // FIXME: FLTK is not consistent in how it sends FL_PUSH/FL_RELEASE
+    // events:
+    //   * https://github.com/fltk/fltk/issues/1076
```

Comment:
This only seems to be FLTK 1.4, and we use 1.3.

Do we still need this workaround for 1.3? It's not apparent why.